doc/index.rst

```diff
@@ -215,7 +215,13 @@ Release Process
 Timeframe
 ---------
 
-.. todo:: Update with information about the cycle_build
+Our timeframe for deploying code to the summit is on an approximately bimonthly cycle.
```

Comment:
I think this is mostly related to new cycles. Revisions can be done at any time. Maybe clarify this? A new cycle means, changes to xml/sal/salobj...

Reply:
Do you want to clarify that this is related to cycle updgrades?